src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -101,6 +103,7 @@ public void Load(string settingsXml)
             TargetPlatform = GetInnerText(runConfiguration, "TargetPlatform");
             TargetFrameworkVersion = GetInnerText(runConfiguration, "TargetFrameworkVersion");
             TestAdapterPaths = GetInnerText(runConfiguration, "TestAdapterPaths");
+            CollectSourceInformation = GetInnerTextAsBool(runConfiguration, "CollectSourceInformation", true);
```

Comment:
`CollectSourceInformation` is true by default to allow for older VS to work seamlessly, they will never set the setting.